modeltestSDK/resources.py

```diff
@@ -187,20 +217,38 @@ class DataPoints(Resource):
     def __len__(self):
         return len(self.time)
 
+    @property
     def timeseries(self):
-        return self.client.timeseries.get_by_id(self.timeseries_id)
+        if self.client:
+            return self.client.timeseries.get_by_id(self.timeseries_id)
+        else:
+            return None
 
-    def plot(self, **kwargs):  # pragma: no cover
+    def plot(self, show: bool = True, **kwargs):  # pragma: no cover
```

Comment:
Med show-flagget kan "no cover" kanskje fjernes i testene

Reply:
Agreed. updated locally, will push when you're done